Makefile

```diff
@@ -24,39 +36,27 @@ include build/make/static-analysis.mk
 include build/make/clean.mk
 include build/make/digital-signature.mk
 
-# ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
-ENVTEST_K8S_VERSION = 1.23
-K8S_INTEGRATION_TEST_DIR=${TARGET_DIR}/k8s-integration-test
-K8S_UTILITY_BIN_PATH=$(WORKDIR)/.bin
-K8S_RESOURCE_YAML=$(TARGET_DIR)/${ARTIFACT_ID}_${VERSION}.yaml
 
 # Setting SHELL to bash allows bash commands to be executed by recipes.
 # This is a requirement for 'setup-envtest.sh' in the test target.
 # Options are set to exit when a recipe line exits non-zero or a piped command fails.
 SHELL = /usr/bin/env bash -o pipefail
 .SHELLFLAGS = -ec
 
-.PHONY: all
-all: build
+##@ EcoSystem
 
-##@ General
+.PHONY: build
+build: docker-build image-import k8s-apply ## Builds a new version of the setup and deploys it into the K8s-EcoSystem.
```

Comment:
This target name should change as it does more than just building

Reply:
I tried to stay as close to the cesapp names as possible. In this case `build` builds the new version and also deploys it directly.

Reply:
yeah, but what if the cesapp names were not really good either :D I think we can do better here; now is the time to improve!